### PR TITLE
VM and VMSS licenseType description update, allows value to be updated

### PR DESCRIPTION
**Pull request by Sandido**
Work for https://msazure.visualstudio.com/One/_workitems/edit/8323663 
The description for the LicenseType property in the VM and VMSS properties indicate the property cannot be updated. This is not true. The value can be updated to 'Windows_Server' or 'Windows_Client' without issue for Windows OS resources. 

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [x] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by varunkumta** [comment]
not sure why this change needs Arm review - removing label

**Review by ArcturusZhang — APPROVED** [review]

**Comment by ArcturusZhang** [comment]
Hi @Sandido is this good to merge?

**Comment by Sandido** [comment]
This PR is good to be merged.

**Review by grizzlytheodore — APPROVED** [review]
Looks good to me. Please confirm with API owner on the property behavior

**Comment by varunkumta** [comment]
removing label again and updating checklist in PR description - unchecked requirement for api review

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json [hunk 1]
```diff
@@ -9040,7 +9040,7 @@
         },
         "licenseType": {
           "type": "string",
-          "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Minimum api-version: 2015-06-15"
```
Comment (dochung4) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: This needs to be changed for other previous API versions as well. Alternatively, we could go with having the updated description for the upcoming API version (2020-12-01) and onwards, leaving the old API versions untouched.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json [hunk 2]
```diff
@@ -9040,7 +9040,7 @@
...
+          "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/hybrid-use-benefit-licensing) <br><br> Minimum api-version: 2015-06-15"
```
Comment (smotwani) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: @rosingla needs to review this as we are now supporting AHUB for Linux VMs and we need to include the possible values for the license values support for Linux VMs and point to the reference for the more details.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json (hunk 1 above)
Comment (Sandido) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: I plan to change it for the current version and the 2020-12-01 version.

File: specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json (hunk 2 above)
Comment (rosingla) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: Suggested Description (same needs to be updated at all other places i.,e all other descriptions for licenseType field in this file) :

Specifies that the image or disk that is being used was licensed on-premises.

Possible values for Windows Server operating system are:

Windows_Client

Windows_Server

Possible values for Linux Server operating system are:

RHEL_BYOS

SLES_BYOS

For more information:

Azure Hybrid Use Benefit for Windows Server

Azure Hybrid Use Benefit For Linux Server

Minimum api-version: 2015-06-15

@smotwani / Abhishek/ Mayank to review and ack.

Comment (smotwani) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: ack

Comment (rosingla) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json:9043]: acked by Mayank as well

Comment (Sandido) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json]: @rosingla , @smotwani , I made these suggested changes.

**Comment by Sandido** [comment]
Ready to merge.

**Pull request merged**